tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier/Program.cs

```diff
@@ -59,5 +56,23 @@ public void Main(string[] args)
                 throw;
             }
         }
+
+        // Copied from DNX's DnuEnvironment.cs
+        private string GetHome()
+        {
+#if DNX451
+            return Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
+#else
+            if (_runtimeEnv.OperatingSystem == "Windows")
+            {
+                return Environment.GetEnvironmentVariable("USERPROFILE") ??
+                    Environment.GetEnvironmentVariable("HOMEDRIVE") + Environment.GetEnvironmentVariable("HOMEPATH");
+            }
+            else
+            {
+                return Environment.GetEnvironmentVariable("HOME");
```

Comment:
FYI this might not always be set, see https://github.com/dotnet/corefx/pull/2889. I guess it's time to expose this from CoreFX :smile: